### PR TITLE
Added models for the application

### PR DESCRIPTION
**Pull request by UzunIon**

### CODE REVIEW

File: app/Models/Activity.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        'restaurant_id',
+        'activity_type',
+        'description',
+        'price',
+        'duration',
+    ];
+
+    public function restaurant()
+    {
+        return $this->belongsTo('Restaurant');
```
Comment (ghost) [inline app/Models/Activity.php]: please specify full class path `Restaurant::class`, don't forget about import

File: app/Models/Activity.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    }
+    public function tours()
+    {
+        return $this->belongsToMany('Tour');
```
Comment (ghost) [inline app/Models/Activity.php]: please specify full classpath `Tour::class`, don't forget about the import

File: app/Models/Activity.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    }
+}
```
Comment (ghost) [inline app/Models/Activity.php:28]: please pay attention on fi warning

File: app/Models/ContactInfo.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        'phone_number',
+        'email',
+        'gender',
+    ];
+
+    public $timestamps = false;
+
+    public function customer()
+    {
+        return $this->belongsTo('Customer');
```
Comment (ghost) [inline app/Models/ContactInfo.php]: please specify full classpath `Customer::class`, don't forget about the import

Comment (ghost) [reply app/Models/ContactInfo.php]: here you should replace relation with direct `hasOne`
https://laravel.com/docs/8.x/eloquent-relationships#one-to-one

File: app/Models/Customer.php [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+class Customer extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'name',
+        'surname',
+    ];
+
+    public function contactinfo()
```
Comment (ghost) [inline app/Models/Customer.php]: `contactInfo`

File: app/Models/Tour.php [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+    protected $fillable = [
+        'name',
+        'details',
+        'guest_house_id',
+        'location_id',
+        'sport_activity_id',
+    ];
+
+    public function guesthouses()
```
Comment (ghost) [inline app/Models/Tour.php]: `guestHouses`

File: app/Models/Tour.php [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    {
+        return $this->hasMany('GuestHouse');
```
Comment (ghost) [inline app/Models/Tour.php]: try to find where should be placed inversed relation and where direct
https://laravel.com/docs/8.x/eloquent-relationships#one-to-many

File: app/Models/Tour.php [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+    }
+    public function locations()
+    {
+        return $this->hasMany('Location');
```
Comment (ghost) [inline app/Models/Tour.php]: try to find where should be placed inversed relation and where direct
https://laravel.com/docs/8.x/eloquent-relationships#one-to-many

File: app/Models/Tour.php [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+    }
+    public function sportactivities()
+    {
+        return $this->hasMany('SportActivity');
```
Comment (ghost) [inline app/Models/Tour.php]: try to find where should be placed inversed relation and where direct
https://laravel.com/docs/8.x/eloquent-relationships#one-to-many

File: app/Models/Tour.php [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+    }
+    public function activities()
+    {
+        return $this->belongsToMany('Activity');
```
Comment (ghost) [inline app/Models/Tour.php]: Use class name `Activity::class` for this, don't forget to import

**Review by ghost — CHANGES_REQUESTED** [review]

File: app/Models/Activity.php (hunk 1 above)
Comment (UzunIon) [reply app/Models/Activity.php]: Added class path

File: app/Models/Activity.php (hunk 2 above)
Comment (UzunIon) [reply app/Models/Activity.php]: Added class path

File: app/Models/ContactInfo.php (hunk 4 above)
Comment (UzunIon) [reply app/Models/ContactInfo.php]: Added class path, and fixed the relation

File: app/Models/Customer.php (hunk 5 above)
Comment (UzunIon) [reply app/Models/Customer.php]: Fixed method name

File: app/Models/Tour.php (hunk 6 above)
Comment (UzunIon) [reply app/Models/Tour.php]: Fixed method name

File: app/Models/Tour.php (hunk 7 above)
Comment (UzunIon) [reply app/Models/Tour.php]: Fixed the method name and relation

File: app/Models/Tour.php (hunk 8 above)
Comment (UzunIon) [reply app/Models/Tour.php]: Fixed the method name and relation

File: app/Models/Tour.php (hunk 9 above)
Comment (UzunIon) [reply app/Models/Tour.php]: Fixed the class path, relation and method name for this relation

File: app/Models/Tour.php (hunk 10 above)
Comment (UzunIon) [reply app/Models/Tour.php]: Fixed

**Pull request merged**